r/NEWS.md

```diff
@@ -26,7 +26,7 @@
   * String operations: `strsplit()` and `str_split()`; `strptime()`; `paste()`, `paste0()`, and `str_c()`; `substr()` and `str_sub()`; `str_like()`; `str_pad()`; `stri_reverse()`
   * Date/time operations: `lubridate` methods such as `year()`, `month()`, `wday()`, and so on
   * Math: logarithms (`log()` et al.); trigonometry (`sin()`, `cos()`, et al.); `abs()`; `sign()`; `pmin()` and `pmax()`; `ceiling()`, `floor()`, and `trunc()`
-  * Conditional: `ifelse()` and `if_else()` (fixed-precision decimal numbers do not yet work and factors/dictionaries are converted to character strings); `case_when()` (currently works with numeric data types but not character strings, factors/dictionaries, or lists/structs); `coalesce()` (lists/structs do not work and factors/dictionaries are converted to character strings)
+  * Conditional: `ifelse()` and `if_else()` (fixed-precision decimal numbers not yet implemented and factors/dictionaries are converted to character strings); `case_when()` (currently works with numeric data types but not character strings, factors/dictionaries, or lists/structs); `coalesce()` (lists/structs not yet implemented and factors/dictionaries are converted to character strings)
```

Comment:
Slightly more concise/less repetitive about not implemented:

```suggestion
  * Conditional functions, with some limitations on input type in this release: `ifelse()` and `if_else()` for all but `Decimal` types; `case_when()` for only numeric types; `coalesce()` for all but lists/structs. Note also that in this release, factors/dictionaries are converted to strings in these functions.
```

Reply:
Thanks! I tweaked this slightly to reflect that `case_when()` supports logical and temporal types too.